src/Runtime/OMInstrument.inc

```diff
@@ -181,7 +181,7 @@ void OMInstrumentPoint(const char *opName, int64_t tag, const char *nodeName) {
     return;
 
   // Print header
-  printf("#%3d) %s %s", instrumentCounter,
+  printf("#%d) %s %s", instrumentCounter,
```

Comment:
This is to remove padded spaces when the value is 1 or 2 digits, which simplifies post-processing of the printed output, like using `awk`.